src/ansible_navigator/configuration_subsystem/navigator_post_processor.py

```diff
@@ -647,7 +647,15 @@ def _normalize_path(
         exit_messages: list[ExitMessage] = []
 
         if entry.value.current is not C.NOT_SET:
-            normalized = Path(entry.value.current).resolve(strict=strict_resolve).as_posix()
+            try:
+                normalized = Path(entry.value.current).resolve(strict=strict_resolve).as_posix()
+            except FileNotFoundError:
```

Comment:
To avoid the coverage error, for this particular case you can use `# pragma: no cover`